thetis/options.py

```diff
@@ -452,6 +452,8 @@ class CommonModelOptions(FrozenConfigurable):
         None, allow_none=True, help="Source term for 2D tracer equation").tag(config=True)
     horizontal_diffusivity = FiredrakeCoefficient(
         None, allow_none=True, help="Horizontal diffusivity for tracers").tag(config=True)
+    tracer_only = Bool(
+        False, help="Hold shallow water variables in initial state and consider tracer advection in the associated (constant) velocity field").tag(config=True)
```

Comment:
Maybe it's better to add this in `ModelOptions2d` as this is not yet implemented in the 3D solver.

Reply:
Good point!